twilight-model/src/guild/onboarding/prompt_type.rs

```diff
@@ -0,0 +1,66 @@
+use serde::{Deserialize, Serialize};
+
+/// The type of an onboarding prompt.
+#[derive(Clone, Copy, Debug, Deserialize, Eq, Hash, PartialEq, Serialize)]
+#[non_exhaustive]
+#[serde(from = "u8", into = "u8")]
+pub enum OnboardingPromptType {
+    /// A prompt that allows the user to select multiple options.
+    MultipleChoice,
+    /// A prompt that allows the user to use a dropdown to select an option.
+    DropDown,
+    /// An unknown prompt type.
+    Unknown(u8),
+}
+
+impl OnboardingPromptType {
+    pub const fn name(self) -> &'static str {
```

Comment:
This should probably be renamed to `kind` to be consistent with other methods on enums